.ado/publish.yml

```diff
@@ -56,22 +50,31 @@ jobs:
           script: |
             VERSION=$(node .ado/get-next-semver-version.js)
             echo "Set version to: $VERSION"
-        condition: and(succeeded(), ne(variables['Build.SourceBranchName'], 'main'))
+        condition: and(succeeded(), ne(variables['Build.SourceBranch'], 'refs/heads/main'))
+  
+      - task: CmdLine@2
+        displayName: Set latest tag if latest stable branch
+        inputs:
+          script: |
+            LATEST=true
+            echo "Set latest to: $LATEST"
+        condition: and(succeeded(), ne(variables['Build.SourceBranch'], 'refs/heads/main'))
 
       - task: CmdLine@2
-        displayName: Set release type
+        displayName: Set release type to 'nightly'
         inputs:
           script: |
-            RELEASE_TYPE=$(node .ado/get-next-release-type.js)
+            RELEASE_TYPE=nightly
             echo "Set release type to: $RELEASE_TYPE"
+        condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
   
       - task: CmdLine@2
-        displayName: Set latest tag if latest stable branch
+        displayName: Set release type to 'release'
```

Comment:
if i am reading this right, release type is still nightly in line 75?

Reply:
Yes. Looks like I messed up. I'll fix this.